prog/anderson_model.h

```diff
@@ -112,12 +82,19 @@ class anderson_model: public quantum_model {
 
       mpi_cout << "Terms with 4 operators" << std::endl;
       Lat.printTerms(4);
-    };
+    }
   }
+
+  struct {
```

Comment:
is this a new recommended way of encapsulating parameters into structs and fetching them with std::get?

Reply:
This structure is essentially a namespace that isolates args option objects from eponymous data members (c.f. `anderson_model::levels` and `anderson_model::args_options.levels`).